test/integration/opcache-php56/serverspec/opcache-php56_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'serverspec'
+
+set :backend, :exec
+
+describe command 'echo "<?php phpinfo(); ?>" | php' do
+  its(:stdout) { should match /PHP Version.*5.6/ }
+  its(:stdout) { should match /opcache\.memory_consumption.+1024/ }
+  its(:stdout) { should match /opcache\.max_accelerated_files.+1000/ }
```

Comment:
Why not check all of the options you set either by default or also set via other means? Same thing for below.

Reply:
fixed